graph/schema.resolvers.go

```diff
@@ -36,8 +44,8 @@ func (r *queryResolver) Vehicle(ctx context.Context, id int) (*model.Vehicle, er
 }
 
 // Dcn is the resolver for the dcn field.
-func (r *queryResolver) Dcn(ctx context.Context, node []byte) (*model.Dcn, error) {
-	return r.Repo.GetDCNByNode(ctx, node)
+func (r *queryResolver) Dcn(ctx context.Context, by model.DCNBy) (*model.Dcn, error) {
+	return r.Repo.GetDCN(ctx, by)
```

Comment:
Can we put more logic here? I'd also like to check that `node` has length 32.

Reply:
https://github.com/DIMO-Network/identity-api/pull/38#discussion_r1331956449

Reply:
Are you checking the length?

Reply:
It was more about not checking the length in the resolver.

Reply:
Are we checking it anywhere?

Reply:
Yeah, will check in the resolver